tests/test_reqif.py

```diff
@@ -440,27 +440,29 @@ def test_create_requirements_attributes_with_default_values(
         assert isinstance(attr, reqif.AbstractRequirementsAttribute)
 
     @pytest.mark.parametrize(
-        "type_hint,value,xml",
+        "type_hint,value,xml,expected_repr_name",
         [
-            pytest.param("Int", 0, None),
-            pytest.param("Int", 1, "1"),
-            pytest.param("Str", "", None),
-            pytest.param("Str", "test", "test"),
-            pytest.param("Float", 0.0, None),
-            pytest.param("Float", 1.0, "1.0"),
-            pytest.param("Date", None, None),
+            pytest.param("Int", 0, None, "[Integer Value Attribute]"),
+            pytest.param("Int", 1, "1", "[Integer Value Attribute]"),
+            pytest.param("Str", "", None, "[String Value Attribute]"),
+            pytest.param("Str", "test", "test", "[String Value Attribute]"),
+            pytest.param("Float", 0.0, None, "[Real Value Attribute]"),
+            pytest.param("Float", 1.0, "1.0", "[Real Value Attribute]"),
+            pytest.param("Date", None, None, "[Date Value Attribute]"),
```

Comment:
The `expected_repr_name` only correlates with the `type_hint`, and is completely independent from the other two parameterized arguments. It might be worth to get it out into its own test case, to prevent even more repetition from creeping in in the future.

And once we have that, we can even go and test the longer `type_hint`s now. :)